spec/services/sla_breach_service_spec.rb

```diff
@@ -222,4 +222,102 @@
       end
     end
   end
+
+  describe ".outgoing_interaction_breaches" do
+    let(:vita_partner_1) { create(:organization) }
+    let(:vita_partner_2) { create(:organization) }
+
+    context "processing on a Friday @ 10:05am UTC" do
+      before do
+        t = Time.utc(2021, 2, 6, 10, 5) # 2/6/21, Saturday
+        Timecop.freeze(t.prev_occurring(:friday)) # 2/5/21, Friday
+        # breaches at vita_partner_1
+        client1 = create(:client, vita_partner_id: vita_partner_1.id, tax_returns: [create(:tax_return, status: 'prep_ready_for_prep')]) # breach
+        client1.update(first_unanswered_incoming_interaction_at: t.prev_occurring(:monday), last_interaction_at: nil)
+        # breaches at vita_partner_2
+        client2 = create(:client, vita_partner_id: vita_partner_2.id, tax_returns: [create(:tax_return, status: 'prep_ready_for_prep')]) # breach
+        client2.update(first_unanswered_incoming_interaction_at: 6.days.ago, last_interaction_at: 7.days.ago)
+
+        client3 = create(:client, vita_partner_id: vita_partner_2.id, tax_returns:  [create(:tax_return, status: 'prep_ready_for_prep')]) # breach
+        client3.update(first_unanswered_incoming_interaction_at: t.prev_occurring(:monday), last_interaction_at: 15.years.ago, )
+        # not in breach
+        client4 = create(:client, first_unanswered_incoming_interaction_at: t.prev_occurring(:monday), last_interaction_at: t, vita_partner_id: vita_partner_2.id,  tax_returns: [create(:tax_return, status: 'prep_ready_for_prep')]) # no breach
+        client4.update(first_unanswered_incoming_interaction_at: t.prev_occurring(:monday), last_interaction_at: t)
```

Comment:
it looks like this line updates properties that were already set on the line above